.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": "eslint:recommended",
+	"env": {
+		"es6": true,
+		"node": true
+	},
+	"rules": {
+		"no-mixed-spaces-and-tabs": "off",
+		"no-unused-vars": ["error", {"argsIgnorePattern": "^_"}]
```

Comment:
Could you explain this line? I'm not familiar with what it's doing here

Reply:
There is some case when you need to get and use just second argument and first one will be shown as unused and displayed as error. ex:
```
array.map((value, index) => console.log(index))
```
and in this case `value` is unused. so by prefixing it with underscore `_value` will not mark it as error.